gnomad/utils/transcript_annotation.py

```diff
@@ -177,93 +181,67 @@ def tissue_expression_ht_to_array(
-    :param filter_to_genes: List of genes to filter to.
-    :param filter_to_csqs: List of consequence terms to filter to.
+    :param ht: Table of transcript expression information.
+    :param filter_to_genes: Optional list of genes to filter to. Default is None.
+    :param match_by_gene_symbol: Whether to match by gene symbol instead of gene ID.
+        Default is False.
+    :param filter_to_csqs: Optional list of consequences to filter to. Default is None.
     :param ignore_splicing: If True, ignore splice variants. Default is True.
-    :param filter_to_homs: If True, filter to variants with at least one
-        homozygote in `freq` field. Default is False.
-    :param filter_to_protein_coding: If True, filter to protein coding
-        transcripts. Default is True.
-    :param vep_annotation: Name of annotation in 'vep' annotation,
-        one of the processed consequences: ["transcript_consequences",
-        "worst_csq_by_gene", "worst_csq_for_variant",
-        "worst_csq_by_gene_canonical", "worst_csq_for_variant_canonical"].
-        For example, if you want to annotate each variant with the worst
-        consequence in each gene it falls on and the transcript expression,
-        you would use "worst_csq_by_gene". Default is "transcript_consequences".
-    :return: Table of variants that fall on transcripts of interest.
+    :param filter_to_protein_coding: Whether to filter to protein coding transcripts.
+        Default is True.
+    :param vep_root: Name used for root VEP annotation. Default is 'vep'.
+    :return: Table of transcript expression information prepared for annotation.
     """
     # TODO: Filter to only CDS regions?
-    if "freq" not in list(ht.row_value):
-        logger.info(
-            "No 'freq' field found in input Table. You cannot use the filter_to_homs"
-            " option."
-        )
-        ht = ht.select(vep=ht.vep)
-        ht = process_consequences(ht)
-        ht = ht.select(vep_processed_csqs=ht.vep[vep_annotation])
-    else:
-        ht = ht.select(freq=ht.freq, vep=ht.vep)
-        ht = process_consequences(ht)
-        ht = ht.select(freq=ht.freq, vep_processed_csqs=ht.vep[vep_annotation])
-
-    # Explode the processed transcript consequences to be able to key by
-    # transcript ID
-    ht = ht.explode(ht.vep_processed_csqs)
 
-    if filter_to_genes:
-        ht = ht.filter(
-            hl.literal(filter_to_genes).contains(ht.vep_processed_csqs.gene_id)
-        )
-
-    if filter_to_csqs:
-        ht = ht.filter(
-            hl.literal(filter_to_csqs).contains(
-                ht.vep_processed_csqs.most_severe_consequence
-            )
-        )
-
-    # TODO: Need to modify process consequences to ignore splice variants,
-    #  because these can occur on intronic regions?
-    splicing = hl.literal(
-        ["splice_acceptor_variant", "splice_donor_variant", "splice_region_variant"]
-    )
+    splice_csqs = [
```

Comment:
Maybe we should add this as a global utils.vep? similar to the others like `CSQ_CODING_HIGH_IMPACT`, `CSQ_CODING_MEDIUM_IMPACT`, `CSQ_CODING_LOW_IMPACT`, ...